packages/discovery-react-components/src/components/StructuredQuery/messages.ts

```diff
@@ -0,0 +1,86 @@
+export interface Messages {
+  /**
+   * override the default text about satisfying all/any of the rules
+   * Use %dropdown% as a placeholder in the text to specify where the dropdown should appear
+   */
+  satisfyRulesDropdownText: string;
+  /**
+   * override the default text for the 'all' dropdown option
+   */
+  satisfyRulesDropdownAllOptionText: string;
+  /**
+   * override the default text for the 'any' dropdown option
+   */
+  satisfyRulesDropdownAnyOptionText: string;
+  /**
+   * override the default label text used as the textual representation of what the satisfy rules dropdown is for
+   */
+  satisfyRulesDropdownLabelText: string;
+  /**
+   * override the default placeholder text for the Field dropdown
+   */
+  fieldDropdownPlaceholderText: string;
+  /**
+   * override the default text for the Field dropdown title
+   */
+  fieldDropdownTitleText: string;
+  /**
+   * override the default placeholder text for the Operator dropdown
+   */
+  operatorDropdownPlaceholderText: string;
+  /**
+   * override the default text for the Operator dropdown title
+   */
+  operatorDropdownTitleText: string;
+  /**
+   * override the default text for the Operator dropdown 'is' option
+   */
+  operatorDropdownIsOptionText: string;
+  /**
+   * override the default text for the Operator dropdown 'is not' option
+   */
+  operatorDropdownIsNotOptionText: string;
+  /**
+   * override the default text for the Operator dropdown 'contains' option
+   */
+  operatorDropdownContainsOptionText: string;
+  /**
+   * override the default text for the Operator dropdown 'does not contain' option
+   */
+  operatorDropdownDoesNotContainOptionText: string;
+  /**
+   * override the default text for the Value input label text
+   */
+  valueInputLabelText: string;
+  /**
+   * override the default text for the Value input placeholder text
+   */
+  valueInputPlaceholderText: string;
+  /**
+   * override the default text for the button to add a rule
+   */
+  addRuleText: string;
+  /**
+   * override the default text for the button to add a group of rules
+   */
+  addGroupRulesText: string;
+}
+
+export const defaultMessages: Messages = {
+  satisfyRulesDropdownText: 'Satisfy %dropdown% of the following rules',
```

Comment:
i think the syntax that other libraries use for variable interpolation is `{dropdown}` -> 
1. i18n next -> https://www.i18next.com/translation-function/interpolation
1. icu message syntax -> https://formatjs.io/guides/message-syntax/

Reply:
All right, will update to use the brackets 👍

Reply:
we can call this out in our contributing guidelines as well